prepare.sh

```diff
@@ -55,7 +55,12 @@ if [[ "$PROFILE_NAME" == centos-stream-* ]]; then
 
     CENTOS_STREAM_RELEASE=$(sed 's/centos-stream-//' <<< "$PROFILE_NAME")
     EPEL_RELEASE_PACKAGE_URL="https://dl.fedoraproject.org/pub/epel/epel-release-latest-$CENTOS_STREAM_RELEASE.noarch.rpm"
-    rpm -q epel-release && yum -y reinstall "$EPEL_RELEASE_PACKAGE_URL" || yum -y install "$EPEL_RELEASE_PACKAGE_URL"
+    if [[ "$CENTOS_STREAM_RELEASE" == "9" ]]; then
```

Comment:
This should distinguish between epel9 and epel9-next dist-git branch. I think there could also be a complication with epel9-next being pre-installed for the former case.